### ISSUE TITLE
Retrain the model but the loss always comes to 'nan', anything wrong with my step?

### ISSUE THREAD

**Issue by beatbox505** [ayziksha/DSIN#18]
Thank you for your great work in DSC compression!
When I try to retrain the DSIN model using KITTI Stereo, the loss always goes into 'nan'. I download the pretrained weights and put it under src/weights/, then I change the path of 'load_model_name' in ae_config, set the 'train_model', 'load_model' true. I try to set True or False for 'AE_only', the loss decrease when i just only train the AE, but the loss become 'nan' when i train the whole model. I am not sure which setting is wrong. Hope you can help me.
